spec/requests/api/v1/cocktail/rating_request_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+RSpec.describe 'Rating API' do
+  describe 'show response endpoint' do
+    before :all do
+      User.destroy_all
+      Cocktail.destroy_all
+      @cocktail_1 = create(:cocktail)
+      @user_1 = create(:user)
+    end
+
+    it 'can send rating of a specific cocktail' do
+      post "/api/v1/cocktails/#{@cocktail_1.id}/rating", params: {
+        user_id: @user_1.id,
+        cocktail_id: @cocktail_1.id,
+        stars: 3
+      }
+
+      expect(response.status).to eq(201)
+
+      cocktail_1_rating = JSON.parse(response.body, symbolize_names: true)
+      rating = Rating.find(cocktail_1_rating[:data][:id])
+
+      expect(cocktail_1_rating).to have_key(:data)
+      expect(cocktail_1_rating[:data]).to be_a Hash
+
+      expect(cocktail_1_rating[:data][:type]).to eq("cocktail_rating")
+      expect(cocktail_1_rating[:data][:attributes][:user_id]).to eq(rating.user_id)
+      expect(cocktail_1_rating[:data][:attributes][:cocktail_id]).to eq(rating.cocktail_id)
```

Comment:
Question: Did we consider the first rating for a cocktail here? We will theoretically need to check for and create a new cocktail if the cocktail is not yet in our DB, right? Otherwise, every new cocktail would risk throwing this error?

Reply:
I don't see any change in the code in response to your question, @mkrumholz. Did you two discuss this?

Reply:
we actually may end up not using this... or maybe it'll change.